src/lib/dom.js

```diff
@@ -0,0 +1,20 @@
+export function waitForElement (selector) {
```

Comment:
Is this used? Do we need it?

Reply:
Here: https://github.com/ipfs-shipyard/ipfs-webui/blob/b25792935a3d34f0f2c3ee29310d72c44df5105d/src/bundles/files.js#L55